client/my-sites/themes/themes-banner/photo-blog.jsx

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+
+import { localize } from 'i18n-calypso';
+import ThemesBanner from './index';
+
+class PhotoBlogBanner extends PureComponent {
+	render() {
+		const { translate } = this.props;
+		return (
+			<ThemesBanner
+				themeId={ 'photo-blog' }
```

Comment:
For string literals in JSX, double quotes are preferred over curly-braces-and-single-quotes, i.e. `themeId="photo-blog"` (same for `themeName`).

Reply:
(And also for `backgroundColor`, `image`, and `imageTransform`.)

Reply:
Good to know!